ui/apps/platform/src/Containers/NetworkGraph/simulation/UploadYAMLButton.tsx

```diff
@@ -0,0 +1,37 @@
+import { Button, FileUpload } from '@patternfly/react-core';
+import { FileUploadIcon } from '@patternfly/react-icons';
+import React from 'react';
+
+type UploadYAMLButtonProps = {
+    onFileInputChange: (
+        _event: React.ChangeEvent<HTMLInputElement> | React.DragEvent<HTMLElement>,
+        file: File
+    ) => void;
+};
+
+function UploadYAMLButton({ onFileInputChange }: UploadYAMLButtonProps) {
+    return (
+        <>
+            <Button
```

Comment:
The PF docs did show me a component for file upload https://www.patternfly.org/v4/components/file-upload. The only thing is that it doesn't look quite like the mock. I decided to use a regular button that, once clicked, triggers clicking the file upload component. If this doesn't seem like something we want to do, I can just use the file upload component even if it is different from the mocks. @alanonthegit @zhenpesky  let me know your thoughts on that approach

Reply:
@alanonthegit @zhenpesky Linda also mentioned we have this approach for uploading files that we have in the policy management section 

<img width="1440" alt="Screenshot 2022-12-20 at 11 37 23 AM" src="https://user-images.githubusercontent.com/4805485/208752079-62a7ea2a-f4f5-41c3-a4d8-804cd4319481.png">

Reply:
@sachaudh got it. It works for me!

Reply:
Which one works for you? @alanonthegit

Reply:
Oh I misread. I thought they were the same option.
for option 1, clicking the button bypasses the modal and it goes into the upload Mac finder window?

Reply:
Yessir

Reply:
We talked through slack and agreed to keep it how it is in this PR